Mobile/starter_project_dartsmiths/pubspec.yaml

```diff
@@ -59,10 +59,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
```

Comment:
[very-nit] it might be better to also get rid of the instruction comment 
or modify it to look like it is describing what is already done

Reply:
These are files flutter generates so we can fix these to describe our things more but I don't see the point in him changing this or even in getting rid of them.

Reply:
These are files flutter generates so we can fix these to describe our things more but I don't see the point in him changing this or even in getting rid of them.